src/docs/button.mdx

```diff
@@ -17,7 +17,7 @@ import { Button } from "../index";
 
 ### Normal
 
-<Playground>
+<Playground style={{ backgroundColor: "#eee", margin: '-2em', padding: '2em', width: 'auto' }}>
```

Comment:
Oops, completely unintentional. @bgptr I can't repro the dot bug. Which browser are you in?

Reply:
The screenshot was taken in chrome on Linux, but it doesn't perfect in Firefox either. 
In ff it's good by default, but after clicking it looks like this:
<img width="97" alt="image" src="https://user-images.githubusercontent.com/52497040/143481650-89815ea7-5dcd-418e-8306-413ba2d26a5d.png">


Google Chrome: Version 96.0.4664.45 (Official Build) (64-bit)
Firefox: 94.0.2

Reply:
Weird, can't repro. Look:

https://user-images.githubusercontent.com/13955303/143489875-f00a3dba-80d3-47b0-b6a3-aae06b55a3c6.mp4

Tried both firefox and chrome with multiple screen sizes. Only thing I can think of is related to the browser font-size since we are using `rems`

Reply:
Ohh, probably you are right. The browser font size is the default, but my dpi is set to 144 in my `.Xresources` file. 
I set it long ago and didn't notice anything broken yet though.

Reply:
hmmm

Reply:
I checked it on other platforms, and it looks good indeed.

Reply:
Could repro. Will probably go with pixels for small units.